frontend/components/customizer/path-builder.tsx

```diff
@@ -153,10 +155,22 @@ export default function PathBuilder({
     })
   }
 
-  function renderLiteral(literal: string, tokenIndex: number) {
+  function renderLiteral(
+    literal: string,
+    tokenIndex: number,
+    onlyLiterals: boolean
```

Comment:
It's a bit odd to have a parameter `onlyLiterals` on a function called `renderLiteral()`.

```suggestion
    pathContainsOnlyLiterals: boolean
```


Alternatively you could invert it and call it `hasNamedParams`.